tensorflow/core/kernels/fused_batch_norm_op.cc

```diff
@@ -1554,6 +1648,22 @@ class FusedBatchNormGradOpV3 : public FusedBatchNormGradOpBase<Device, T, U> {
   }
 };
 
+template <typename Device, typename T, typename U>
+class FusedBatchNormGradOpEx : public FusedBatchNormGradOpBase<Device, T, U> {
+  static constexpr bool kWithSideInputAndActivation = true;
```

Comment:
I think originally we separate the PR to two commits for (1) Add _FusedBatchNormGradEx op (This is with the test on this new op.) and (2) Add remapper support. I merged them together in one commit in this PR since I think they have been reviewed and approved.
For the XLA lowering, our XLA team plans to add this op to their allowlist. Probably another PR. For this PR, we simply don't remap to this new op when XLA is detected to avoid XLA seeing it.